source/_docs/frontend/browsers.markdown

```diff
@@ -29,7 +29,7 @@ We would appreciate if you help to keep this page up-to-date and add feedback.
 
 | Browser               | Release        | State      | Comments                 |
 | :-------------------- |:---------------|:-----------|:-------------------------|
-| [Safari]              |                | works      |                          |
+| [Safari]              |                | works      | Map has corrupted CSS.   |
```

Comment:
https://github.com/home-assistant/home-assistant-polymer/issues/345
https://github.com/home-assistant/home-assistant/issues/6827
https://community.home-assistant.io/t/map-page-problems/14501
https://www.reddit.com/r/homeassistant/comments/62oxmn/broken_map/

Don't know, sorry :(...
According to links above, the problem also exists on Edge.